app/src/main/java/org/mozilla/focus/utils/HomeTileUtils.java

```diff
@@ -0,0 +1,28 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.utils;
+
+import android.content.Context;
+import org.json.JSONObject;
+import java.io.InputStream;
+
+public class HomeTileUtils {
+    public static JSONObject loadDefaultTiles(Context context) {
+        String json;
+        try {
+            InputStream is = context.getAssets().open("defaults/DefaultTiles.json");
```

Comment:
Same: hard-coded string

But actually, since this is a util, you probably don't want to put the string into the util at all (though you're welcome to make the argument for it!). I would say the caller should be in charge of doing this (because you utils to be generic), and because this will eventually used by different locales.